.github/workflows/go-test.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ "ubuntu", "windows", "macos" ]
-        go: [ "1.16.x" ]
+        go: [ "1.15.x", "1.16.x" ]
```

Comment:
This will cause CI tests to fail since there is code in the repo that requires go1.16.  Either remove go1.15.x here or make the code-base compatible with go1.15.x (there are only a couple of lines to change).

Reply:
This is due to the `io.ReadAll` call in *client/client.go*. Two options:
1. Use the backwards-compatible `ioutil.ReadAll`.
2. Wait until Go 1.17 is released (which is when we drop Go 1.15 support).